### DISCUSSION TITLE
Documentation for fcitx5-chinese-addons

### DISCUSSION THREAD

**Discussion by lesbianlizard** [fcitx/fcitx5#359] [General]
Does the [fcitx5-chinese-addons](https://github.com/fcitx/fcitx5-chinese-addons) repository have user documentation somewhere? I'm learning Mandarin, and I'm trying to learn how to use the pinyin addon from that repository as well. I've figured out some of the basic functionality by trial and error, as well as by looking at the settings menu for the pinyin addon, but there are some features mentioned in the settings menu (like prediction, stroke filtering, etc.) that I have no idea how to use.

I don't see any documentation in the repository itself. I found this [Built-in Pinyin](https://fcitx-im.org/wiki/Built-in_Pinyin) page on the wiki, but it doesn't have much information and is really old (links to some Google Code thing).

**Comment by wengxt** [comment]
Yeah, I agree we're currently lacking of user manual on certain features.. updating wiki
